build/charts/theia/provisioning/datasources/create_table.sh

```diff
@@ -257,12 +291,36 @@ clickhouse client -n -h 127.0.0.1 <<-EOSQL
         destinationServicePortName,
         destinationIP;
 
+    {{- if .Values.clickhouse.cluster.enable }}
+    CREATE TABLE IF NOT EXISTS recommendations_local (
+    {{- else }}
     CREATE TABLE IF NOT EXISTS recommendations (
+    {{- end }}
         id String,
         type String,
         timeCreated DateTime,
         yamls String
+    {{- if .Values.clickhouse.cluster.enable }}
+    ) engine=ReplicatedMergeTree('/clickhouse/tables/{shard}/{database}/{table}', '{replica}')
+    {{- else }}
     ) engine=MergeTree
+    {{- end }}
     ORDER BY (timeCreated);
-    
+
+    {{- if .Values.clickhouse.cluster.enable }}
```

Comment:
An idea about maintainability: can we add comment lines to the script explaining what we are doing in the various steps?

Reply:
Add some comments, thanks!